index.php

```diff
@@ -196,10 +196,19 @@ function smf_main()
 	if (!empty($topic) && empty($board_info['cur_topic_approved']) && !allowedTo('approve_posts') && ($user_info['id'] != $board_info['cur_topic_starter'] || $user_info['is_guest']))
 		fatal_lang_error('not_a_topic', false);
 
-	$no_stat_actions = array('clock', 'dlattach', 'findmember', 'jsoption', 'likes', 'loadeditorlocale', 'modifycat', 'requestmembers', 'smstats', 'suggest', 'about:unknown', '.xml', 'xmlhttp', 'verificationcode', 'viewquery', 'viewsmfile');
-	call_integration_hook('integrate_pre_log_stats', array(&$no_stat_actions));
 	// Do some logging, unless this is an attachment, avatar, toggle of editor buttons, theme option, XML feed etc.
-	if (empty($_REQUEST['action']) || !in_array($_REQUEST['action'], $no_stat_actions))
+	$no_stat_actions = array(
+		'clock', 'dlattach', 'findmember', 'jsoption', 'likes', 'loadeditorlocale',
+		'modifycat', 'requestmembers', 'smstats', 'suggest', 'about:unknown',
+		'.xml', 'xmlhttp', 'verificationcode', 'viewquery', 'viewsmfile',
+	);
+	$no_stat_areas = array(
+		'profile' => 'alerts_popup',
+	);
```

Comment:
Might be better as a multi-dimensional array here.

Reply:
An excellent idea.